src/Card/README.md

```diff
@@ -334,6 +334,62 @@ When using horizontal variant Paragon provides additional component `Card.Body`
 </>
 ```
 
+### Card Content Block Empty
+
+With image
```

Comment:
Should these be treated as sub-headings, e.g. `#### With image`?

Reply:
Fixed